tests/generate_packageTest.py

```diff
@@ -0,0 +1,99 @@
+import os
+import shutil
+import sys
+import unittest
+from unittest.mock import patch
+from bears.generate_package import (touch, create_file_from_template,
+                                    create_file_structure_for_packages,
+                                    perform_upload, main,
+                                    create_upload_parser)
+
+
+class touchTest(unittest.TestCase):
+
+    def test_file_doesnt_exist(self):
+        if os.path.exists('TestFile.py'):
+            os.remove('TestFile.py')
+        self.assertFalse(os.path.exists('TestFile.py'))
+        touch('TestFile.py')
+        self.assertTrue(os.path.exists('TestFile.py'))
+
+    def tearDown(self):
+        os.remove('TestFile.py')
```

Comment:
Now that you have created a teardown (which is good!), will you still need the os.remove in line `16`?